### ISSUE TITLE
could not build under Mac OSX Big Sur

### ISSUE THREAD

**Issue by drzhouq** [WerWolv/ImHex#145]
follow the recipe detailed on the readme page. at the last step `make -j` encountered an error: ImHex/include/helpers/disassembler.hpp:3:10: fatal error: 'capstone/capstone.h' file not found, though I already added these two lines on the
export LDFLAGS="-L/usr/local/opt/luajit-openresty/lib -L/usr/local/opt/libarchive/lib -L/usr/local/opt/libffi/lib -L/usr/local/opt/capstone/lib -L/usr/local/opt/libmagic/lib"
export CPPFLAGS="-I/usr/local/opt/luajit-openresty/include -I/usr/local/opt/libffi/include -I/usr/local/opt/libarchive/include -I/usr/local/opt/capstone/include -I/usr/local/opt/libmagic/include"

**Comment by WerWolv** [comment]
Interesting! That's the error that started showing up in the CI too out of nowhere. I thought it was only an issue there but I guess not. Is there no `capstone/capstone.h` file in `/usr/local/opt/capstone/include` on Mac anymore?

**Comment by jonnyhoff** [comment]
Same issue.

```
[ 96%] Building CXX object CMakeFiles/imhex.dir/source/views/view_strings.cpp.o
[ 98%] Building CXX object CMakeFiles/imhex.dir/source/views/view_tools.cpp.o
In file included from /Users/me/tools/ImHex/source/views/view_disassembler.cpp:1:
In file included from /Users/me/tools/ImHex/include/views/view_disassembler.hpp:5:
/Users/me/tools/ImHex/include/helpers/disassembler.hpp:3:10: fatal error: 'capstone/capstone.h' file not found
```

**Comment by drzhouq** [comment]
@WerWolv Thanks for responding. I did check those files are there (both the header and libraries).
find /usr/local/opt/capstone/include -type f
/usr/local/opt/capstone/include/capstone/capstone.h
....

**Comment by WerWolv** [comment]
Ugh brew why...
It sets the include path to `/usr/local/Cellar/capstone/4.0.2/include/capstone`. Of course if won't find `/usr/local/Cellar/capstone/4.0.2/include/capstone/capstone/capstone.h` then...

Should be an easy fix at least. Thanks!

**Comment by WerWolv** [comment]
Should be fixed in the latest master :)

**Comment by drzhouq** [comment]
@WerWolv many thanks for your quick response. It seems to ignore the environmental variable CPPFLAGS. How did you identify the problem so fast?

**Comment by WerWolv** [comment]
It's `CXXFLAGS` with cmake afaik.
Also that error existed for weeks before in the CI and I had no idea how to fix it because I thought the reason was GitHub Actions. I ended up just printing the path cmake is putting in the include paths and saw that it points to `capstone/include/capstone` instead of just `capstone/include` like literally anywhere else.

**Comment by drzhouq** [comment]
cool @WerWolv I did add the CXXFLAGS but it didn't solve the issue. Anyhow, I am able to build (found out I could have just downloaded your nightly built, I was blind). I was hoping to use the tool to reverse engineer a data file (binary) generated by a legacy program so that I can programmatically produce data files for testing. I have used vim -d file1 file2 to compare, but just one single parameter change causes a lot of changes in the generated data files. I am not sure where to start with ImHex. It seems it does not have the comparison tool?
